demo/index.html

```diff
@@ -84,8 +84,9 @@
 </body>
 
 <!-- ES6-related imports -->
-<script src="/node_modules/angular2/bundles/angular2-polyfills.js"></script>
 <script src="/node_modules/es6-shim/es6-shim.min.js"></script>
+<script src="/node_modules/zone.js/dist/zone.js"></script>
+<script src="/node_modules/reflect-metadata/Reflect.js"></script>
 <script src="/node_modules/systemjs/dist/system.js"></script>
```

Comment:
After moving to Angular RC and cleaning up node_module was getting issue in browser. Adding zone.js and Reflect.js resolves the problem.